core/src/test/java/org/elasticsearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -135,6 +135,13 @@ public static String blockMasterFromFinalizingSnapshot(final String repositoryNa
         return masterName;
     }
 
+    public static String blockMasterFromCreatingSnapshot(final String repositoryName) {
```

Comment:
The method name made me think that it would prevent the master from creating a snapshot at all. Maybe we can call it something along the lines of "blockMasterFromFinalizingSnapshot"?